scss/_variables.scss

```diff
@@ -210,7 +210,7 @@ $escaped-characters: (
   ("#","%23"),
 ) !default;
 
-// Selector that could break a selector stack cross-browser compatibility
+// Selectors which are isolated in the transition mixin to prevent invalid selector stack
```

Comment:
@XhmikosR This is more precise, inspired by the above `$escaped-characters` map's comment.

LGTY?